### PR TITLE
Automatically port System.Text.Json.JsonException

### PR DESCRIPTION
**Pull request by carlossanlop**
New batch.

### CODE REVIEW

File: xml/System.Text.Json/JsonException.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
-﻿<Type Name="JsonException" FullName="System.Text.Json.JsonException">
-  <TypeSignature Language="C#" Value="public class JsonException : Exception" />
+﻿<Type Name="JsonException" FullName="System.Text.Json.JsonException"><TypeSignature Language="C#" Value="public class JsonException : Exception" />
```
Comment (rpetrusha) [inline xml/System.Text.Json/JsonException.xml]: This change should be reverted.

File: xml/System.Text.Json/JsonException.xml [hunk 2]
```diff
@@ -31,7 +30,7 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Creates a new exception object to relay error information to the user.</summary>
```
Comment (rpetrusha) [inline xml/System.Text.Json/JsonException.xml]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Text.Json.JsonException" /> class.</summary>
```

File: xml/System.Text.Json/JsonException.xml [hunk 3]
```diff
@@ -51,8 +50,8 @@
         <Parameter Name="message" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="message">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="message">The context specific error message.</param>
+        <summary>Creates a new exception object to relay error information to the user.</summary>
```
Comment (rpetrusha) [inline xml/System.Text.Json/JsonException.xml]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Text.Json.JsonException" /> class with a specified error message.</summary>
```

File: xml/System.Text.Json/JsonException.xml [hunk 4]
```diff
@@ -51,8 +50,8 @@
         <Parameter Name="message" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="message">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="message">The context specific error message.</param>
```
Comment (rpetrusha) [inline xml/System.Text.Json/JsonException.xml]:
```suggestion
        <param name="message">The context-specific error message.</param>
```

File: xml/System.Text.Json/JsonException.xml [hunk 5]
```diff
@@ -97,9 +96,9 @@
         <Parameter Name="innerException" Type="System.Exception" />
       </Parameters>
       <Docs>
-        <param name="message">To be added.</param>
-        <param name="innerException">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="message">The context specific error message.</param>
+        <param name="innerException">The exception that caused the current exception.</param>
+        <summary>Creates a new exception object to relay error information to the user.</summary>
```
Comment (rpetrusha) [inline xml/System.Text.Json/JsonException.xml]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Text.Json.JsonException" /> class. with a specified error message and a reference to the inner exception that is the cause of this exception.</summary>
```

File: xml/System.Text.Json/JsonException.xml [hunk 6]
```diff
@@ -97,9 +96,9 @@
         <Parameter Name="innerException" Type="System.Exception" />
       </Parameters>
       <Docs>
-        <param name="message">To be added.</param>
-        <param name="innerException">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="message">The context specific error message.</param>
```
Comment (rpetrusha) [inline xml/System.Text.Json/JsonException.xml]:
```suggestion
        <param name="message">The context-specific error message.</param>
```

File: xml/System.Text.Json/JsonException.xml [hunk 7]
```diff
@@ -263,7 +258,7 @@ Note that the `bytePositionInLine` counts the number of bytes (that is, UTF-8 co
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets a message that describes the current exception.</summary>
         <value>To be added.</value>
```
Comment (rpetrusha) [inline xml/System.Text.Json/JsonException.xml]:
```suggestion
        <value>The error message that describes the current exception.</value>
```

File: xml/System.Text.Json/JsonException.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.Text.Json/JsonException.xml]: I don't know what happened here. I will revert it.

Comment (carlossanlop) [reply xml/System.Text.Json/JsonException.xml]:
```suggestion
<Type Name="JsonException" FullName="System.Text.Json.JsonException">
  <TypeSignature Language="C#" Value="public class JsonException : Exception" />
```

File: xml/System.Text.Json/JsonException.xml (hunk 5 above)
Comment (carlossanlop) [reply xml/System.Text.Json/JsonException.xml]: I made a minor edit to your suggestion:
 . -> ,

File: xml/System.Text.Json/JsonException.xml [hunk 8]
```diff
@@ -1,4 +1,4 @@
-﻿<Type Name="JsonException" FullName="System.Text.Json.JsonException">
```
Comment (carlossanlop) [inline xml/System.Text.Json/JsonException.xml]: Not sure why this line remains changed, it's probably the BOM character. I don't think it will matter.

**Comment by rpetrusha** [comment]
I don't think it matters either, @carlossanlop. I'll merge now.

**Pull request merged**